### PR TITLE
Updated the start date for the "abbrviated list of release note" section in the README file

### PR DESCRIPTION
**Pull request by yzluka**
#7592 
Edited line 121 of the README file. 

This edit will update the last revision date of the abbreviated list of release note. 

Signed-off-by: Yixin Zhang <zhany217@wfu.edu>

### CODE REVIEW

**Comment by jjhursey** [comment]
ok to test

File: README [hunk 1]
```diff
@@ -118,7 +118,7 @@ The rest of this README file contains:
 ===========================================================================
 
 The following abbreviated list of release notes applies to this code
-base as of this writing (March 2017):
+base as of this writing (March 2019):
```
Comment (jjhursey) [inline README:121]: We should adjust the month marker from `March` to `April` too since the other PR will be merged in April.

Comment (yzluka) [reply README:121]: yes, and I guess also change the year to 2020 if there are modifications in other sections of the list?

Comment (yzluka) [reply README]: both month and year are adjusted now

Comment (jjhursey) [reply README]: Awesome. Thanks.

**Pull request merged**